lib/firmware/app_layout.c

```diff
@@ -496,6 +496,50 @@ void layout_ethereum_address_notification(const char *desc, const char *address,
     layout_notification_icon(type, &sp);
 }
 
+/*
+ * layout_nano_address_notification() - Display nano address notification
+ *
+ * INPUT
+ *     - desc: description of address being shown (normal or multisig)
```

Comment:
Does Nano have multisig?

Reply:
No it doesn't. Didn't notice that part when I was adapting the comment block for the Nano method. Will update that block.

Reply:
Fixed in b739a6b